### DISCUSSION TITLE
Saving session problem

### DISCUSSION THREAD

**Discussion by Touexe** [Davincible/goinsta#11] [Q&A]
i tried defer insta.Export(filepath) to the filepath i refer to but it doesn't seem to appear in my destination folder. the question is should i put the folder path or filepath but what extension anyway. tried .json but reusing it doesn't work, everything i requested to instagram after importing the session gave nothing.

**Comment by Davincible** [comment]
You put the file path relative to the current working directory. File extension makes no difference, but .json can be convenient if you want to view it. if you just pass "goinsta.json" as argument you see a file in the directory you ran the command from?

**Reply by Touexe** [reply]
yes! after passing a file with .json extension i can see it in the directory. however, reusing the session doesn't give me any result after i'm trying to fetch feed or conversation

**Reply by Davincible** [reply]
The session only saves your cookies and device information, not any content info such as posts or DMs. After importing a session file you will have to fetch them from scratch. You can do this with `insta.OpenApp()`, this will fetch everything. Alternatively you can fetch the specific content you need manually
